### ISSUE TITLE
Load picoFeed without composer

### ISSUE THREAD

**Issue by bloute** [fguillot/picoFeed#247]
Hi ! I'm almost newbie in PHP so excuse the simplicity of my question, but how do we load picoFeed without composer ? Do we have to include a specific file ?
Actually, I have the following structure : 
- index.php
-- lib
--- PicoFeed
---- Reader

When I do "use PicoFeed\Reader\Reader;" in index.php, I got the following error :  Class 'PicoFeed\Reader\Reader' not found

Could you please help me ?

Many thanks.

**Comment by BernhardPosselt** [comment]
You need to require **every** file that is needed before you use the class. Which is not what you want.

You simply download the composer.phar (see https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx) and create the autoloader once by executing this command once in the picofeed directory (see https://getcomposer.org/doc/03-cli.md#dump-autoload):

    php composer.phar dump-autoload

**Comment by BernhardPosselt** [comment]
Ah right, forget that command, you need the dependencies, so instead of the above command simply use:

    php composer.phar require

**Comment by BernhardPosselt** [comment]
TL;DR composer is easy to use, no need to install anything and much simpler and faster than manually including hundreds of files ;)

**Comment by bloute** [comment]
Ok, many thanks for you ultra quick answer. It works great like that !
